### PR TITLE
Problem: Gravity Bridge Documentation is outdated

### LINKED ISSUE #527: Problem: Gravity Bridge Documentation is Outdated
**Describe the bug**

Gravity Bridge Documentation is Outdated:
- Missing keystore information
- Repetitive (WET)
- Does not documentation to run gorc as a systemd service

**To Reproduce**
Read the current documentation

**Expected behavior**
Documentation should be clear and adhere to DRY principles

**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

### PR DESCRIPTION
**Pull request by mofhusseini**
- Add setup-gorc-service.sh shell script to set up systemd service
  - Refactor docs into small focused parts
  - Add keystores docs
  - Rename files for clarity

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [x] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [x] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

closes #527

### CODE REVIEW

**Review by leejw51crypto — APPROVED** [review]
lgtm

File: docs/gravity-bridge/dev-setup-guide.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ You need to prepare two accounts for the orchestrator, one for ethereum and one
 
 ### Creating the config:
 
-We will use the below `config.toml` to specify the directory where the keys will be generated and some of the configs needed to run the orchestrator. Creating a `gorc.toml` file in the same directory as `gorc` and paste the following config:
+We will use the below `config.toml` to specify the directory where the keys will be generated and some of the configs needed to run the orchestrator. Create a `gorc.toml` file in the same directory as `gorc` and paste the following config:
 
 
 ```toml
```
Comment (thomas-nguy) [inline docs/gravity-bridge/dev-setup-guide.md:42]: maybe add as a comment 

# keystore TO BE UPDATED to "Aws" if we choose to AWS as key storage management

Comment (thomas-nguy) [reply docs/gravity-bridge/dev-setup-guide.md:42]: nvm it is explained after

**Comment by thomas-nguy** [comment]
Overall LGTM

One thing that may be good to clarify is that if you run "orchestrator", then you don't need to run a "relayer" separately because the orchestrator gorc will run both orchestrator and relayer (unless you use the flag --orchestrator-only)

**Review by thomas-nguy — APPROVED** [review]
lgtm

**Review by yihuang — APPROVED** [review]

**Pull request merged**